mautic/mautic
# Report query for debug mode

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:

If you use dev environment you can see  query on report's detail page. This PR added support also for enabled debug mode. This allow you print query for fast check on production enviroment.


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Check report detail page on production
3. Check same report detail page with enabled debug = true in local.php. You should see sql queries on the bottom of reports 

![image](https://user-images.githubusercontent.com/462477/184862133-d71c97d1-afc3-48bb-b575-1e8d790f35f9.png)





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11397"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
